pages/deployment/aws.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Deploy Memgraph on AWS EC2 instance 
+description: Learn how to deploy your Memgraph instance on the AWS EC2 instance.
+---
+
+import { Card, Cards, Tabs, Tab } from 'nextra/components'
+
+# Deploy Memgraph in AWS EC2 instance
+
+This guide will show you how to deploy Memgraph on an AWS EC2 instance. The
+guide will cover only the specific bits that are different from the general
+deployment guide you can find for native [linux](./linux.mdx) or
+[docker](./docker.mdx) deployments.
+
+This guide assumes you have an AWS account and are familiar with the AWS
+console. If you are not, you can follow the [AWS EC2
+documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html).
+
+
+## Crating a new EC2 instance 
+
+The first step is to have an EC2 instance running on AWS, where you will deploy
+Memgraph. Here are some guidelines for opening a new EC2 instance for memgraph. 
+
+### Picking the OS 
+
+When creating the EC2 instance, you need to pick the OS you wish to have on the
+instance. If you are going to use the Memgraph native installation, you need to
+pick a Linux distribution that Memgraph supports.
+
+Memgraph supports the following Linux distributions:
+
+- Ubuntu (20.04, 22.04)
+- Fedora (38, 39)
+- Debian (11, 12) 
+- Centos (7, 9) 
+- Amazon Linux (2)
+- RedHat (7, 9)
+- Rocky (9.3)
+
+This way, you will manage the native installation of Memgraph, which can bring
+some of the speed benefits. 
+
+If you are going to use the Memgraph Docker image, you can pick any Linux
+distribution you prefer, as long as you can run Docker on it.
+
+
+### Picking the instance type
+
+When creating the EC2 instance, you need to pick the instance type. The process
+highly depends on your actuall workload. For heavy graph processing, consider
+using instances with more CPU and memory. 
+
+Memgraph, by default, stores all data in the RAM, if you are going to use the
+default storage mode (`IN_MEMORY_TRANSACTIONAL` or `IN_MEMORY_ANALYTICAL`), make
+a calculation [how much data you will
+need]((/fundamentals/storage-memory-usage#calculate-storage-memory-usage)).
```

Comment:
```suggestion
If you are running Memgraph in `IN_MEMORY_TRANSACTIONAL` (default mode) or in `IN_MEMORY_ANALYTICAL` storage mode, all data is stored in RAM. That means it would be good to first calculate [how much memory you will
need](/fundamentals/storage-memory-usage#calculate-storage-memory-usage).
```